controllers/dev_null_logger.go

```diff
@@ -0,0 +1,19 @@
+package controllers
```

Comment:
@dee0sap Feel free to make this aware of the log level given to the controller and be able to set the level. Give it the logger when you create it, so it can log to the controller's log.